build-support/build.gradle.kts

```diff
@@ -20,3 +20,7 @@ gradlePlugin {
     }
   }
 }
+
+dependencies {
+  implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31")
```

Comment:
Can this file refer to `buildSrc`? It'd be nice to keep the kotlin version in the `Dependencies.kt`
Here wouldn't be a place people would look at to update versions

Reply:
Unfortunately, it doesn't look like so, when I tried, `deps` was not resolving.

Reply:
I didn't realize `build-support` was its own project. ~Maybe we could just move the dependencies over there and get rid of `buildSrc`... that'd be a follow-up~ that didn't work.